init.lua

```diff
@@ -7,9 +7,13 @@ ts_furniture.kneeling_bench = minetest.settings:get_bool("ts_furniture.kneeling_
 
 -- Used for localization
 local S = minetest.get_translator("ts_furniture")
+local has_more_player_monoids = minetest.get_modpath("more_player_monoids")
+local has_player_monoids = minetest.get_modpath("player_monoids")
+
+local is_sitting_by_player_name = {}
```

Comment:
Needs removal upon player leave

Reply:
i missed this last year, and someone just poked me to fix it now. anyone got a good way to create recurring reminders for such things?

Reply:
Perhaps check the status of your PRs at https://github.com/pulls every once in a while?